### PR TITLE
community challenges in the wiki

### PR DESCRIPTION
**Pull request by merwanehamadi**
### Background

We want to be able to give a link to people so they can :
- understand what challenges are
- browse  challenges
- create challenges
- run challenges and submit the solution
### PR Quality Checklist
- [X] My pull request is atomic and focuses on a single change.
- [X] I have thoroughly tested my changes with multiple different prompts.
- [X] I have considered potential risks and mitigations for my changes.
- [X] I have documented my changes clearly and comprehensively.
- [X] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -127,22 +127,3 @@ When you run Pytest locally:
             - Or: The test might be poorly written. In that case, you can make suggestions to change the test.
 
 In our CI pipeline, Pytest will use the cassettes and not call paid API providers, so we need your help to record the replays that you break.
-
-
-### Community Challenges
```
Comment (ntindle) [inline CONTRIBUTING.md]: Leave a mention of it here as a reference to them

**Comment by ntindle** [comment]
I get a 404 for all the links in the sidebar

File: CONTRIBUTING.md (hunk 1 above)
Comment (PortlandKyGuy) [reply CONTRIBUTING.md]: @merwanehamadi , I agree with @ntindle that it would be good to leave a reference to the new page.

**Comment by p-i-** [comment]
This is a mass message from the AutoGPT core team.
Our apologies for the ongoing delay in processing PRs.
This is because we are re-architecting the AutoGPT core!

For more details (and for infor on joining our Discord), please refer to:
  https://github.com/Significant-Gravitas/Auto-GPT/wiki/Architecting

**Review by PortlandKyGuy — APPROVED** [review]

**Review by p-i- — COMMENTED** [review]
❤️

**Pull request merged**